### PR TITLE
Update the specification page with a link to new PDF draft

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: src/_guides/language/spec.md [hunk 1]
```diff
@@ -9,26 +9,29 @@ For a gentler introduction to Dart, see the
 
 ## Dart 2
 
+### Latest completed specification
```
Comment (kwalrath) [inline src/_guides/language/spec.md]: Why do we need this section head? It seems unnecessary.

File: src/_guides/language/spec.md [hunk 2]
```diff
@@ -9,26 +9,29 @@ For a gentler introduction to Dart, see the
...
-* Feature specifications in the dart-lang/sdk repo:
-  * [docs/language/informal](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
-* Feature specifications in the dart-lang/language repo:
-  * [accepted](https://github.com/dart-lang/language/tree/master/accepted)
-  * [working](https://github.com/dart-lang/language/tree/master/working)
-* Dart language specification work in progress:
-  * [Source file (LaTeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
-  * [Community-contributed PDF built from the source file](https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/dartLangSpec.pdf)
+The in-progress formal Dart language specification is written using
+the [LaTeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex),
```
Comment (kwalrath) [inline src/_guides/language/spec.md]: Let's rephrase this slightly (and get rid of the excess paren):

...is written as [a LaTeX file](...)

File: src/_guides/language/spec.md [hunk 3]
```diff
@@ -9,26 +9,29 @@ For a gentler introduction to Dart, see the
...
+and is available as a [draft specification in PDF format](https://dart-specification.firebaseapp.com/DartLangSpecDraft.pdf).
```
Comment (kwalrath) [inline src/_guides/language/spec.md]: Do you really want to maintain a separate site for this? I could publish it on dartlang.org.

File: src/_guides/language/spec.md (hunk 1 above)
Comment (mit-mit) [reply src/_guides/language/spec.md]: Removed

File: src/_guides/language/spec.md (hunk 2 above)
Comment (mit-mit) [reply src/_guides/language/spec.md]: Done

File: src/_guides/language/spec.md (hunk 3 above)
Comment (mit-mit) [reply src/_guides/language/spec.md]: Hosting on dartlang.org is not an option given how we deploy that, and where the source for this PDF is located

**Pull request merged**